engines/bastion_katello/app/assets/javascripts/bastion_katello/products/details/repositories/details/views/repository-info.html

```diff
@@ -310,7 +317,16 @@ <h4 translate>Content Counts</h4>
           </td>
         </tr>
 
-        <tr ng-show="repository.content_type === 'yum'">
+        <tr ng-show="repository.content_type === 'deb'">
```

Comment:
I think we're already showing deb packages under the Content Types - so I'm seeing it show up twice in the table on the repository details view

Reply:
Typical case of rebase-hell ;-)

Reply:
IMHO it makes no sense to show `deb Packages` count after `Errata` count, so I vote for removing the existing location of the `deb Packages` count and move it to this location. Even though this sorta messes up the order of table-rows, because right now they are sorted by `content_type`.
A solution to this would be to move `deb Packages` further up, to be the first table-row in `<tbody>`. But `errata` count would still be within the "yum-section".

Reply:
Agree, Errata should come after the deb Packages!